tests/test_setup_bind_mounts

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+. "${srcdir:-.}/common.sh"
+
+cat >"$TMP/snap.name.app" <<EOF
+@unrestricted
+EOF
+
+cat >"$TMP/snap.name.fstab" <<EOF
+/src /dst none bind 0 0
+EOF
+
+printf "Test that bind mounts work "
+"$L" snap.name.app /bin/true
```

Comment:
Thanks, this test is actually bogus :/ I think we need a proper spread test here, its a bit tricky because `snapd` does not have the full support for writing the fstab file yet, so I will ponder a bit if I can fabricate stuff or if I will just wait for snapd.